### PR TITLE
halloween update, add ghosts to cammie 'faces'tm

### PR DESCRIPTION
**Pull request by ajuvercr**
This adds funny ghosts on top of some faces on cammie.
This takes 100% cpu because it is client side, but you can turn it of.
Face detection isn't really that great, puts a lot of ghosts at walls etc, but it is still fun.

### CODE REVIEW

File: content/assets/scripts/halloween/worker.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+var face_cascade, eye_cascade;
+
+function loadFaceDetectTrainingSet() {
+	if (face_cascade == undefined) {
+		face_cascade = new cv.CascadeClassifier();
+		let load = face_cascade.load('../../test/data/haarcascade_frontalface_default.xml');
```
Comment (wschella) [inline content/assets/scripts/halloween/worker.js]: Shouldn't this file be committed as well?

File: content/assets/scripts/halloween/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<html>
```
Comment (wschella) [inline content/assets/scripts/halloween/index.html]: Is this file needed?

**Comment by wschella** [comment]
The CI should resolve if you delete the test file, the codeclimate issue doesn't really matter.

File: content/assets/scripts/halloween/worker.js (hunk 1 above)
Comment (ajuvercr) [reply content/assets/scripts/halloween/worker.js]: That html file was obsolete, just like some code in worker.js, I removed everything that was not required with the latest commit.

**Comment by werthen** [comment]
Check out the preview on https://291.pr.zeus.gent/

**Comment by wschella** [comment]
Maybe it's time to put this baby away. It did never really work-work.
